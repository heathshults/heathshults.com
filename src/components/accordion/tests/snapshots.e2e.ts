import { snapIt } from '../../../test';

const component = 'accordion';
const snap = snapIt(component);

snap('emtpy', `<hs-accordion></hs-accordion>`);
snap(
  'two headers and two panes',
  `<hs-accordion>
      <hs-accordion-pane header="Click me">
        This is an accordion
      </hs-accordion-pane>
    </hs-accordion>`
);
snap(
  'two headers and two panes',
  `<hs-accordion>
      <hs-accordion-pane open header="Click me">
        This is an accordion
      </hs-accordion-pane>
    </hs-accordion>`
);
