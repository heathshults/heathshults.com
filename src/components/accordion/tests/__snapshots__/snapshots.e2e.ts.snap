// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`emtpy 1`] = `"<hs-accordion class=\\"hydrated\\"><div class=\\"hs-card hs-card--accordion\\"></div></hs-accordion>"`;

exports[`two headers and two panes 1`] = `
"<hs-accordion class=\\"hydrated\\"><div class=\\"hs-card hs-card--accordion\\"><hs-accordion-pane header=\\"Click me\\" class=\\"hydrated\\"><button role=\\"heading\\" aria-expanded=\\"false\\" class=\\"hs-card__control\\">Click me</button><section aria-hidden=\\"\\" class=\\"hs-card__item hs-card__item--pane\\">
        This is an accordion
      </section></hs-accordion-pane></div></hs-accordion>"
`;

exports[`two headers and two panes 2`] = `
"<hs-accordion class=\\"hydrated\\"><div class=\\"hs-card hs-card--accordion\\"><hs-accordion-pane open=\\"\\" header=\\"Click me\\" class=\\"hydrated\\"><button role=\\"heading\\" aria-expanded=\\"true\\" class=\\"hs-card__control hs-card__control--active\\">Click me</button><section class=\\"hs-card__item hs-card__item--pane\\">
        This is an accordion
      </section></hs-accordion-pane></div></hs-accordion>"
`;
